### DISCUSSION TITLE
What's the difference between this and AdoptOpenJDK for users and developers?

### DISCUSSION THREAD

**Discussion by heftymouse** [microsoft/openjdk#15] [Q&A] [answered]
Will we expect to see performance improvements or new APIs or something?

**Comment by karianna** [comment, marked as answer]
Hi @heftymouse - over time you may well see improvements that benefit our 1st and 3rd party customer workloads on Azure and Microsoft subsidiaries. Although we'll attempt to upstream a majority of those, the risk-averse nature of OpenJDK long-term stable versions (of which Java 11 is one) means that not all of those changes will be accepted upstream.

Each difference will be clearly marked (with the accompanying link/discussion to the attempt to upstream it).

**Reply by heftymouse** [reply]
Got it, thanks a lot!
